tests/library/har.spec.ts

```diff
@@ -248,21 +248,26 @@ it('should include secure set-cookies', async ({ contextFactory, httpsServer },
 it('should include content @smoke', async ({ contextFactory, server }, testInfo) => {
   const { page, getLog } = await pageWithHar(contextFactory, testInfo);
   await page.goto(server.PREFIX + '/har.html');
+  await page.evaluate(() => fetch('/pptr.png').then(r => r.arrayBuffer()));
   const log = await getLog();
 
-  expect(log.entries[0].response.httpVersion).toBe('HTTP/1.1');
-  expect(log.entries[0].response.content.encoding).toBe('base64');
+  expect(log.entries[0].response.content.encoding).toBe(undefined);
   expect(log.entries[0].response.content.mimeType).toBe('text/html; charset=utf-8');
-  expect(Buffer.from(log.entries[0].response.content.text, 'base64').toString()).toContain('HAR Page');
+  expect(log.entries[0].response.content.text).toContain('HAR Page');
   expect(log.entries[0].response.content.size).toBeGreaterThanOrEqual(96);
   expect(log.entries[0].response.content.compression).toBe(0);
 
-  expect(log.entries[1].response.httpVersion).toBe('HTTP/1.1');
-  expect(log.entries[1].response.content.encoding).toBe('base64');
+  expect(log.entries[1].response.content.encoding).toBe(undefined);
   expect(log.entries[1].response.content.mimeType).toBe('text/css; charset=utf-8');
-  expect(Buffer.from(log.entries[1].response.content.text, 'base64').toString()).toContain('pink');
+  expect(log.entries[1].response.content.text).toContain('pink');
   expect(log.entries[1].response.content.size).toBeGreaterThanOrEqual(37);
   expect(log.entries[1].response.content.compression).toBe(0);
+
+  expect(log.entries[2].response.content.encoding).toBe('base64');
+  expect(log.entries[2].response.content.mimeType).toBe('image/png');
+  expect(Buffer.from(log.entries[2].response.content.text, 'base64')).toBeTruthy();
```

Comment:
isn't an empty Buffer also truthy and it would be better to assert the size here?